recipes/bindsnet/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "bindsnet" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 5d81f5502e357bef2a151461110c108a07292b96a16e383b179eba3023e418de
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxext') }}     # [linux]
+  host:
+    - pip
+    - python
+    - poetry
+    - xorg-libxfixes  # [linux]
+  run:
+    - cython
+    - gym >=0.10.4
+    - matplotlib-base
+    - numpy
+    - opencv
+    - pandas
+    - pytest >=3.4.0
+    - python
+    - scikit-image >=0.13.1
+    - scikit-learn >=0.19.1
+    - scipy >=1.1.0
+    - tensorboardx >=1.7
+    - pytorch
+    - torchvision
+    - tqdm
+
+test:
+  imports:
+    - bindsnet
+
+about:
+  home: http://github.com/BINDS-LAB-UMASS/bindsnet
```

Comment:
This should be changed to https://github.com/BindsNET/bindsnet

Reply:
Thanks!